tests/phpunit/Unit/Graph/GraphTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SRF\Tests\Unit\Formats;
+
+use SMW\Test\QueryPrinterRegistryTestCase;
+use SRF\Graph\GraphNode;
+
+
+/**
+ * Tests for the SRF\Graph class.
+ *
+ * @file
+ * @since 1.8
+ *
+ * @ingroup SemanticResultFormats
+ * @ingroup Test
+ *
+ * @group SRF
+ * @group SMWExtension
+ * @group ResultPrinters
+ *
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class GraphTest extends QueryPrinterRegistryTestCase {
+
+	/**
+	 * @see QueryPrinterRegistryTestCase::getFormats
+	 *
+	 * @since 1.8
+	 *
+	 * @return array
+	 */
+	public function getFormats() {
+		return [ 'graph' ];
+	}
+
+	/**
+	 * @see QueryPrinterRegistryTestCase::getClass
+	 *
+	 * @since 1.8
+	 *
+	 * @return string
+	 */
+	public function getClass() {
+		return 'SRF\Graph\GraphPrinter';
+	}
+
+	/**
+	 * Testing class GraphNode
+	 *
+	 * @since 3.0
+	 *
+	 */
```

Comment:
Wrong again. This comment is not needed anyway like so many of the others. Not adding them is much better.

Reply:
see: https://github.com/RacoonDev/SemanticResultFormats/commit/c1acdf0cf7f136a108ae0380959954703d0bbd35